docs/_articles/en/v3.x/salesforce-graph-engine/working-with-sfge.md

```diff
@@ -82,41 +82,36 @@ public class MyClass {
 
 ## "OutOfMemory: Java heap space" Error
 
-The number of paths SFGE needs to create increases exponentially with every conditional or method invocation in your code. As the number of paths explode, SFGE's probability of encountering an OutOfMemory error increases. In addition, a variety of factors such as your OS type, java setup, and other processes running in your machine can influence the heap space assigned by JVM, which then impacts SFGE's execution.
+A number of factors can degrade the Graph Engine's efficiency and increase the probability of encountering an `OutOfMemory` error.
+- With every conditional or method invocation in your code, the number of paths the Graph Engine creates increases exponentially.
+- The Head space assigned by the Java Virtual Machine (JVM) can be influenced by your OS type, Java setup, and the other processes running on your machine.
```

Comment:
@jfeingold35 I think this is one sentence that you updated from the G doc?
"Your OS type, Java setup, and other processes running on your machine can influence the heap space assigned by Java Virtual Machine (JVM)."

Your edit is fine, but is it "head space" as you've written, or "heap space"?

Reply:
Great catch. That should be "heap space".